src/project-tests/scatter-plot-tests.js

```diff
@@ -67,10 +67,38 @@ export default function createScatterPlotTests() {
                 const sortedDots = dots.sort(function(a, b) {
                     return a.getAttribute("data-xvalue") - b.getAttribute("data-xvalue")
                 });
-
-                //check to see if the x locations of the new sorted array are in ascending order
-                for (var i = 0; i < sortedDots.length - 1; ++i) {
-                    FCC_Global.assert.isAtMost(+sortedDots[i].cx.baseVal.value, +sortedDots[i + 1].cx.baseVal.value, "x values don't line up with x locations ");
+                var xAxisQuery = document.querySelector('#x-axis path');
+                //x-axis path x coordinates from d
+                var xAxisLeft = xAxisQuery.getAttribute('d').split(',')[0].split('M')[1]
```

Comment:
Here's an example of code that isn't immediately apparent. What is attribute `d`? Why are we splitting it by the comma and then M? Even an example of the data this is supposed to parse would help. There are more lines like this, but I'm just going to leave this one comment.